data/datastore.yaml

```diff
@@ -1,91 +1,124 @@
-schema_version: v0.5.0
+schema_version: v0.6.0
 dataset_version: v0.1.0
```

Comment:
Should we maybe change the version here to `v1.0.0`? I made a suggestion about what the versioning could refer to here: https://confluence.dmi.dk/display/funwp/Baseline+configurations+for+neural-lam+experiments

Also, I would add a comment at the beginning of the file, or maybe in the `extras` section describing that this is a baseline. I wrote a suggested description on Confluence

> This is our first baseline experiment setup. It was created to be reasonbly fast for training, give reasonable model skill (for the first roll-out step) and leave GPU RAM space (with the 80GB cards we are using) to allow for experimentation in terms of adding variables, levels, increasing size of hidden dimension, etc.

Reply:
@leifdenby you mean the `dataset_version`, right?